Automattic/vip-go-mu-plugins
# Update Jetpack to 7.3

## Description

7.3 has been released and is ready to be deployed to VIP Go sites.

https://github.com/Automattic/jetpack/releases/tag/7.3

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Try frontend + backend requests
1. Try WP CLI commands
1. Verify JP version in WP Admin
